src/core/main.c

```diff
@@ -139,6 +139,11 @@ static sd_id128_t arg_machine_id = {};
 static EmergencyAction arg_cad_burst_action = EMERGENCY_ACTION_REBOOT_FORCE;
 static OOMPolicy arg_default_oom_policy = OOM_STOP;
 
+static struct CPUAffinity {
+        cpu_set_t *set;
+        size_t allocated;
+} arg_cpu_affinity = {};
```

Comment:
one of those days we should probably move the structure definition into process-util.h or so...